templates/huginn/0/rancher-compose.yml

```diff
@@ -0,0 +1,92 @@
+version: '2'
+catalog:
+    name: Huginn
+    version: "0.1"
+    description: Web automation framework. Your agents are standing by!
+    questions:
```

Comment:
I would add catalog questions for DB_USER and DB_NAME.
```
+    - variable: "PUBLISH_PORT"
+        label: "Publish port"
+        description: |
+          Publish external port for Huginn web.
+        required: true
+        type: "int"
+        default: 3000
+      - variable: DB_NAME
+        description: The database name
+        label: Database name
+        required: true
+        default: "huginn"
+        type: string
+      - variable: DB_USER
+        description: The database username
+        label: Database username
+        required: true
+        default: "huginn"
+        type: string
```

Reply:
Done.